src/main/steps/edge-case/address/select/content.test.ts

```diff
@@ -66,7 +69,7 @@ describe('applicant1 > address > select > content', () => {
     const commonContent1 = { language: 'cy', userCase: { applyingWith: 'alone' } } as CommonContent;
 
     const generatedContent1 = generateContent(commonContent1);
-    expect(generatedContent1.section).toBe('Applicant (in Welsh)');
+    expect(generatedContent1.section).toBe('Applicant (in welsh)');
```

Comment:
content should be read from translation. it should be changed to "toBe(cyContent.section)"